web-client/integration-tests/journey/petitionsClerkCreatesNewCaseFromPaper.js

```diff
@@ -254,7 +254,6 @@ export const petitionsClerkCreatesNewCaseFromPaper = (
 
     expect(helper).toMatchObject({
       hasIrsNoticeFormatted: 'No',
-      hasOrders: true,
```

Comment:
Could this be updated to expect some of the new properties that were added to the helper?